test/fuzz/pem/corpus/0

```diff
@@ -0,0 +1,2 @@
+-----BEGIN -----
```

Comment:
>Put the initial corpus into the workdir/corpus directory (in our case examples/png/corpus). Go-fuzz will add own inputs to the corpus directory. Consider committing the generated inputs to your source control system, this will allow you to restart go-fuzz without losing previous work.
Source https://github.com/dvyukov/go-fuzz
https://github.com/dvyukov/go-fuzz-corpus/tree/master/pem/corpus